shell/update_mock_feeds.sh

```
@@ -0,0 +1,41 @@
+# change_price.sh - Script to alter the state of the Bophades Range System on Testnet to mock different scenarios
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)